modules/videoModule/index.js

```diff
@@ -200,6 +211,13 @@ export function PbVideo(videoCore_, getConfig_, pbGlobal_, pbEvents_, videoEvent
   // options: adXml, winner, adUnitCode,
   function loadAdTag(adTagUrl, divId, options) {
     const payload = Object.assign({ adTagUrl }, options);
+
+    if (adQueueCoordinator.requiresQueueing(divId)) {
```

Comment:
I think always queueing would cut down code complexity significantly. By that I mean letting the queue decide if the ad needs to wait or not, instead of forking off two code paths here.
Already I can spot a difference, whether the AD_LOAD_ATTEMPT event fires is now basically random, which makes me wonder how useful it is.

Reply:
I agree

Reply:
@dgirardi addressed